composer.json

```diff
@@ -39,8 +39,8 @@
     },
     "require-dev": {
         "ext-curl": "*",
-        "php-http/client-integration-tests": "dev-phpunit8",
-        "phpunit/phpunit": "^8.5.5",
+        "php-http/client-integration-tests": "^3.0",
```

Comment:
Did dependencies of this package change so that php-http stuff is no longer loaded? That would explain the class not found errors.

Reply:
Oh, it was just me. I thought i released this hours ago but I never clicked on the "submit button". 


And yes, we dropped some HTTPlug stuff to focus on PSR18.